openshift/matomo/openshift/templates/matomo-db/matomo-db-deploy.json

```diff
@@ -22,7 +23,8 @@
           "name": "${NAME}",
           "app": "${APP_NAME}",
           "service": "${NAME}",
-          "template": "matomo-template"
+          "template": "matomo-template",
+          "role": "${NAME}"
```

Comment:
We're getting a bit tag heavy. I feel like we can drop template, and if we're adding role, we can drop service.

Nothing is actually using these tags right now, so it's just adding noise.

Also see below comment re. ${ROLE} - we should probably be using that instead of ${NAME}